ERROR - 2023-05-23 20:17:02 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'muhasin' in C:\xampp\htdocs\muhasin\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\muhasin\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'muhasin', 3306, '', 0)
#1 C:\xampp\htdocs\muhasin\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\muhasin\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\muhasin\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\muhasin\app\Models\ModelLogin.php(19): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\muhasin\app\Controllers\Login.php(32): App\Models\ModelLogin->getDataId('user', Array)
#6 C:\xampp\htdocs\muhasin\system\CodeIgniter.php(927): App\Controllers\Login->prosesLogin()
#7 C:\xampp\htdocs\muhasin\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\muhasin\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\muhasin\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\muhasin\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'muhasin' in C:\xampp\htdocs\muhasin\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\muhasin\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\muhasin\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\muhasin\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\muhasin\app\Models\ModelLogin.php(19): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\muhasin\app\Controllers\Login.php(32): App\Models\ModelLogin->getDataId('user', Array)
#5 C:\xampp\htdocs\muhasin\system\CodeIgniter.php(927): App\Controllers\Login->prosesLogin()
#6 C:\xampp\htdocs\muhasin\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\muhasin\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\muhasin\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\muhasin\system\Commands\Server\rewrite.php(46): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-05-23 20:17:02 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'muhasin'
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Models\ModelLogin.php(19): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Controllers\Login.php(32): App\Models\ModelLogin->getDataId('user', [...])
 5 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Login->prosesLogin()
 6 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 7 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(46): require_once('FCPATH\\index.php')
